backend/controller/sql/queries.sql

```diff
@@ -599,23 +599,30 @@ ON CONFLICT (name, module_id) DO
 UPDATE SET 
   topic_id = excluded.topic_id,
   deployment_id = (SELECT id FROM deployments WHERE key = sqlc.arg('deployment')::deployment_key)
-RETURNING id;
-
--- name: DeleteSubscriptions :exec
+RETURNING 
+  id,
+  CASE 
+    WHEN xmax = 0 THEN true
```

Comment:
What is "xmax"?

Reply:
Chad told me about it, which led me to this post:
https://stackoverflow.com/questions/39058213/differentiate-inserted-and-updated-rows-in-upsert-using-system-columns
It is a bit of black magic though...